Containers/nextcloud/entrypoint.sh

```diff
@@ -229,12 +229,12 @@ if ! [ -f "$NEXTCLOUD_DATA_DIR/skip.update" ]; then
             # php /var/www/html/occ config:app:set updatenotification notify_groups --value="[]"
 
             # Install some apps by default
-            php /var/www/html/occ app:install twofactor_totp
-            php /var/www/html/occ app:install deck
-            php /var/www/html/occ app:install tasks
-            php /var/www/html/occ app:install calendar
-            php /var/www/html/occ app:install contacts
-            php /var/www/html/occ app:install apporder
+            if [ -n "$STARTUP_APPS" ]; then
+                read -ar STARTUP_APPS_ARRAY <<< "$STARTUP_APPS"
+                for app in "${STARTUP_APPS_ARRAY[@]}"; do
+                    php /var/www/html/occ app:install "$app"
+                done
+            fi
```

Comment:
I don't know if this is a problem on my side, but I don't get this loop running

Reply:
What exactly did you do? It will only run this at the very first installation of Nextcloud...

Reply:
okay just tested. Indeed it didnt work. Now it should. Feel free to try again :)

Reply:
I've exported STARTUP_APPS using different ways:
```
export STARTUP_APPS=twofactor_totp deck tasks calendar contacts apporder
export STARTUP_APPS="twofactor_totp deck tasks calendar contacts apporder"
export STARTUP_APPS=(twofactor_totp deck tasks calendar contacts apporder)
```
And then executed the following script
```bash
#!/bin/bash
if [ -n "$STARTUP_APPS" ]; then
read -ra STARTUP_APPS_ARRAY <<< "$STARTUP_APPS"
for app in "${STARTUP_APPS_ARRAY[@]}"; do
echo "$app"
done fi
```
And I never got any output
![grafik](https://user-images.githubusercontent.com/75573284/200672904-d6e62ef9-7d12-434b-8d20-c4af07ca6a9f.png)

Maybe I do something wrong, but I don't know

Reply:
> export STARTUP_APPS="twofactor_totp deck tasks calendar contacts apporder"

can you try to put that at the beginning of the script and try again?

Reply:
![image](https://user-images.githubusercontent.com/42591237/200673562-abac1dea-32da-4b35-9e71-11a524631804.png)

Reply:
![grafik](https://user-images.githubusercontent.com/75573284/200673628-25d7caaf-eb00-41ab-93cd-3f1d9365287d.png)
Made it work

Reply:
Cool!